lib/utils.js

```diff
@@ -65,17 +66,35 @@ function stringifyPropertyName(prop) {
   return typeof prop === "symbol" ? symbolName(prop) : JSON.stringify(propertyName(prop));
 }
 
+function toKey(type, func = "") {
+  return String(func + type).replace(/[./-]+/g, " ").trim().replace(/ /g, "_");
+}
+
 class RequiresMap extends Map {
   constructor(ctx) {
     super();
     this.ctx = ctx;
   }
 
   add(type, func = "") {
-    const key = (func + type).replace(/[./-]+/g, " ").trim().replace(/ /g, "_");
+    const key = toKey(type, func);
+
+    const importPath = (type.includes("/") || type.startsWith(".")) && !path.extname(type) ? type + ".js" : type;
```

Comment:
This&nbsp;checks&nbsp;needs to&nbsp;be&nbsp;updated to&nbsp;account for&nbsp;scoped&nbsp;packages.